File: CPP/Product.cpp
#include "Product.h"

Product::Product(int idProduct, std::string name, std::string brand, double price) {
    this->idProduct = idProduct;
    this->name = name;
    this->brand = brand;
    this->price = price;
}

int Product::getIdProduct() {
    return idProduct;
}

void Product::setIdProduct(int idProduct) {
    this->idProduct = idProduct;
}

std::string Product::getName() {
    return name;
}

void Product::setName(std::string name) {
    this->name = name;
}

std::string Product::getBrand() {
    return brand;
}

void Product::setBrand(std::string brand) {
    this->brand = brand;
}

double Product::getPrice() {
    return price;
}

void Product::setPrice(double price) {
    this->price = price;
}

File: PYTHON/product.py
class Product:
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price

    def getIdProduct(self):
        return self.idProduct

    def setIdProduct(self, idProduct):
        self.idProduct = idProduct

    def getName(self):
        return self.name

    def setName(self, name):
        self.name = name

    def getBrand(self):
        return self.brand

    def setBrand(self, brand):
        self.brand = brand

    def getPrice(self):
        return self.price

    def setPrice(self, price):
        self.price = price
